.github/release.yml

```diff
@@ -0,0 +1,20 @@
+changelog:
+  exclude:
+    authors:
+      - github-actions
+  categories:
+    - title: Features
+      labels:
+        - "type: feature"
+    - title: Bugs
+      labels:
+        - "type: bug"
+    - title: Documentation
+      labels:
+        - "type: docs"
+    - title: Chores
+      labels:
+        - "type: chore"
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
I think we could add enhancement as well or improvements and performance wdyt? Those are things that we often encounter

Reply:
Discussed internally. We will not be adding this right now.